tests/integ/test_secrets.py

```diff
@@ -0,0 +1,128 @@
+#
+# Copyright (c) 2012-2025 Snowflake Computing Inc. All rights reserved.
+#
+
+import pytest
+from snowflake.snowpark.secrets import (
+    get_generic_secret_string,
+    get_username_password,
+    get_secret_type,
+)
+from snowflake.snowpark.types import BooleanType, StringType
+from tests.utils import IS_NOT_ON_GITHUB, RUNNING_ON_JENKINS, Utils
+from snowflake.snowpark import Row
+
+
+@pytest.mark.skipif(
```

Comment:
can we add 2 negative test cases:

1. in github action, we raise not implemented error
2. when a key doesn't exist, assert SQL exception is raised